src/App.js

```diff
@@ -1,12 +1,12 @@
 import React, { Component } from "react";
 import "./App.scss";
+import BusinessesList from "./components/BusinessesList";
 
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <h1>Hello World</h1>
-        <button>Hello</button>
+        <BusinessesList></BusinessesList>
```

Comment:
to avoid conflicts let's remove this from, this in the future would be our router anyway

Reply:
to test components you can add them here, but when you want to push to github remove it from here and then push

Reply:
done